Modules/Scripted/DICOMPlugins/DICOMScalarVolumePlugin.py

```diff
@@ -268,6 +262,17 @@ def examineFiles(self,files):
     for loadable in loadables:
       loadable.files, distances, loadable.warning = DICOMUtils.getSortedImageFiles(loadable.files, self.epsilon)
 
+    # by default, prefer the all-files loadable
+    loadables[0].confidence = .55
+    if subseriesCount == 1 and loadables[0].warning != "":
+      # there was a sorting warning and
+      # only one kind of subseries, so it's probably correct
+      # to make the subseries a higher confidence than the default all-files version
+      for subseriesLoadable in loadables[1:]:
+        if subseriesLoadable.warning == "":
+          subseriesLoadable.confidence = .55
```

Comment:
Is there a specific reason we both increase subseries confidence and decrease loadables confidence? It may be simpler for other DICOM plugins to coexist with the default Scalar Volume plugin if it did not use many different confidence values.

Reply:
It really doesn't matter, because currently we only use the max anyway.  But in the bigger picture I think it's correct to have a below-default confidence in this case.  Someday we may want to use this information for something so it should reflect something about the result of examining the files.

Reply:
OK, then we should keep the confidence value at 0.5 for the loadable that we are most confident about and slightly lower it for the others.

It is better not to increase confidence above 0.5 because that range is quite crowded already (there are plugins that have set their own confidence at slightly higher than 0.5 to take over loading from the default volume reader).

Reply:
Okay, that's fair - the scalarvolume plugin is meant to be the fallback if no other plugins grab the series.